content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
 the top, above the :abbr:`SO (Sales Order)` form. Click the :guilabel:`Delivery` smart button to
 reveal the associated receipt.
 
-.. image:: receipts_delivery_two_steps/two-step-sales-quote.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
```

Comment:
Too zoomed in, would include the larger form/UI

Reply:
Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅